webapp-django/crashstats/status/context_processors.py

```diff
@@ -0,0 +1,18 @@
+from crashstats.status.models import StatusMessage
+
+
+def status_message(request):
+    statuses = (
+        StatusMessage.objects.filter(enabled=True).order_by('-created_at')
+    )
+
+    if not len(statuses):
```

Comment:
By the way, a more efficient way is to do:
```python
if not statuses.count():
```
otherwise it needs to be converted into a list of ORM instance objects in a list but we're only using the length of the list. 

In fact, an even more efficient thing to use is `.exists()`. It's faster in SQL since it returns True as soon as anything exists. 
I.e. 

```
if not statuses.exists():
    return {}
``` 
Not really applicable in this context but might be good to be aware of.